File: Modules/Project/FiyonaMistry/Naukri.com/Web API/NaukriProject/NaukriProject/Models/Job.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

#nullable disable

namespace NaukriProject.Models
{
    public partial class Job
    {
        [Key]
        public int JobId { get; set; }
        public string JobName { get; set; }
        public string JobExperienceNeeded { get; set; }
        public string JobKeySkills { get; set; }
        public string JobSalary { get; set; }
        public string JobLocation { get; set; }
        public string JobDescription { get; set; }
        public string JobRole { get; set; }
        public string JobEmploymentType { get; set; }
        public short? IsOpen { get; set; }
        public int? CompanyId { get; set; }

        //[System.Text.Json.Serialization.JsonIgnore]
        public virtual Company Company { get; set; }
    }
}

File: Modules/Project/FiyonaMistry/Naukri.com/FrontEnd/src/app/shared/Models/Naukri/Job.ts
import { IJob } from '../INaukri/IJob';

export class Job implements IJob{
    jobId? : number;
    jobName : string;
    jobExperienceNeeded : string;
    jobKeySkills : string;
    jobSalary : string;
    jobLocation : string;
    jobDescription : string;
    jobRole : string;
    jobEmploymentType : string;
    isOpen : number;
    companyId : number;

    constructor(id : number, name : string, experience : string, skills : string, salary : string, location : string, desc : string, role : string, employment : string, open : number, company : number){
        this.jobId = id;
        this.jobName = name;
        this.jobExperienceNeeded = experience;
        this.jobKeySkills = skills;
        this.jobSalary = salary;
        this.jobLocation = location;
        this.jobDescription = desc;
        this.jobRole = role;
        this.jobEmploymentType = employment;
        this.isOpen = open;
        this.companyId = company
    }
}